src/include/kernel/tokens.h

```diff
@@ -103,4 +103,8 @@
 /* for backward compatibility */
 #define SOF_TKN_EFFECT_TYPE	SOF_TKN_PROCESS_TYPE
 
+/* Led control for mute switches */
```

Comment:
same comment as for the kernel side, wondering if there should be a MUTE_LED prefix instead of plain vanilla 'LED'.

E.g. on the HifiBerry boards you have an LED that is on why you play something and off in idle.